Extension/c_cpp_properties.schema.json

```diff
@@ -47,6 +47,13 @@
                         ],
                         "description": "If set, it overrides the default mode used by the IntelliSense engine. Windows defaults to msvc-x64 and Linux/Mac default to clang-x64."
                     },
+                    "forceIncludes": {
+                        "description": "A list of files that should be included before any include file in a translation unit",
```

Comment:
I think it's supposed to end in a period?

Reply:
I think it should be "forcedIncludes"? That's what VS calls it.

Reply:
VS calls it "forcedInclude".  ok.